docs/p12-limitation.rst

```diff
@@ -0,0 +1,272 @@
+   ARYPYXNrQGV4YW1wbGUuY29tMIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDF
+   JYrC2pg14dR3pwSxo8OpAHGNbQATXdSsSPGpo8OqvL6OSqTEb+0yO/eaRrl4i4N4
+   zDLVM1Nrd8OqPdNnI57+S7QljGMHcWow4AFC24zY7kFNrFlWjutRAsq/oF1G0ALk
+   IGlxF8viWMZ2Dq5HMnqbJqBz0tuHQ2JIqlO7B0+HrwIDAQABMA0GCSqGSIb3DQEB
+   CwUAA4GBAEg1aNVexn4ghv0dEeyszh1PVK/khOPPRLIiPBrwRQYGUcDI9arAivsV
+   cNAysWzSETBWWxRrI9JqxxSrhNghRQvLJFgTcBBuNwHIaqhq1sAbIdN3rxv22zVD
+   P9ScjuBFg2xXkPBsOTJ7lPCslLJhcCJXBjNQcWDFAUev60QCJAZI
+   -----END CERTIFICATE-----
+
+   # cat intermediate certificate
+   cat int-ca.pem
+   -----BEGIN CERTIFICATE-----
+   MIICbjCCAdegAwIBAgIJAMPIx273oielMA0GCSqGSIb3DQEBCwUAMHAxCzAJBgNV
+   BAYTAkNOMQswCQYDVQQIDAJCSjELMAkGA1UEBwwCQkoxDDAKBgNVBAoMA0FCQzEL
+   MAkGA1UECwwCSVQxEDAOBgNVBAMMB2FiYy5jb20xGjAYBgkqhkiG9w0BCQEWC2Fz
+   a0BhYmMuY29tMB4XDTE4MTAxNjA0MzIyNFoXDTE4MTExNTA0MzIyNFowcDELMAkG
+   A1UEBhMCQ04xCzAJBgNVBAgMAkJKMQswCQYDVQQHDAJCSjEMMAoGA1UECgwDWFla
+   MQswCQYDVQQLDAJJVDEQMA4GA1UEAwwHeHl6LmNvbTEaMBgGCSqGSIb3DQEJARYL
+   YXNrQHh5ei5jb20wgZ8wDQYJKoZIhvcNAQEBBQADgY0AMIGJAoGBAOX8VIM9O4wK
+   QT5KPcs0dVvCzJf944J3ssl+Gaa5WLyuRmYyme5YTST0CjWqDnN8ROR8OgNZi+qF
+   TJHrfk1txlWh91SPV4msKh3c5ZJp9LT/kMGnlgd8blu6FGmFZkBK6tTx7gO5LPME
+   ATvyKi1yYis0pup3AyCQgoJgjwPUWrjdAgMBAAGjEDAOMAwGA1UdEwQFMAMBAf8w
+   DQYJKoZIhvcNAQELBQADgYEAFoTBXkSm9sT1Bd3kWudqYFSQCi+xAUZ2OvDwUnxO
+   m3nO7+U9xX86ULBnYfmeT9ZohXQc44cG630ynBNOfT64/d5yJGbH7+ri/p9SkMXy
+   BVbxQHUHJjA07oyw/gJaFcOkMiSYaRObALxvaXqKFLogS0vaYb+c7P2lfkEtjOvQ
+   07Y=
+   -----END CERTIFICATE-----
+
+   # cat private Key
+   cat server.key
+   -----BEGIN RSA PRIVATE KEY-----
+   MIICXAIBAAKBgQDFJYrC2pg14dR3pwSxo8OpAHGNbQATXdSsSPGpo8OqvL6OSqTE
+   b+0yO/eaRrl4i4N4zDLVM1Nrd8OqPdNnI57+S7QljGMHcWow4AFC24zY7kFNrFlW
+   jutRAsq/oF1G0ALkIGlxF8viWMZ2Dq5HMnqbJqBz0tuHQ2JIqlO7B0+HrwIDAQAB
+   AoGAIfyA2WqZxuAxopb2ZjFXL7FV4g2ib7RDT5gboSUMPEjhiOIxWXP6LijMXJpI
+   qxFSDucU9FAu114EKzsRULyBUgQBbpp8LtcaNGLlgmHr4L7aPlvH/SlhHxdH5tge
+   U2hFhkLr5EDujTwQYcpz8cEnBqz/FtB93qR+BgRX7sjJmKECQQDh7vvn2o8PxMEW
+   MKDvY3auZ/vCixe1FGwmy4nLLE5WKcWFXAf3oq4uRglTKQH1cD7udSKvQ8DPL4Wk
+   WiHnlwS5AkEA32Hbj+VL2atW7ePorSoAzLgqThYB6bG5BymIw4lCpckFCU6YpTIV
+   EGhGDrrgbUlD4h251SHgxnrxAzY1LpOLpwJBAIo14v3rkpan2yKS7vBinSiFzdot
+   snwAmUSGQK38VZOaDA3PxcP0Ta9bArtPm7YkSyselvA2d02HGa73wEPm+2kCQCoj
+   oLKtc7iVLOnlg4AfG1WDLF/coPG/2AK04Bra6tqxaCTQUdVf9D9LHGQs9qdHGeou
+   516AbJGkoZCUikXGCaMCQDoqk+L9VlMiW522vfNwBUfiiRrhQDdUBq+kaEJlqRr/
+   poZy3MRVLQomb7RSyJuXG43SazvEtbdhHt46QP6FHmk=
+   -----END RSA PRIVATE KEY-----
+
+   # bundle certificate, intermediate certificate, and private Key
+   openssl pkcs12 -export -out certificate.pfx -inkey server.key  -in server.crt -certfile int-ca.crt
+   Enter Export Password: your_password
+   Verifying - Enter Export Password: your_password
+
+After generating the PKCS 12 ceritficate certificate.pfx , follow below example steps to extract it.
+
+Example: Extract A PKCS 12 Certificate without Encrypt Private Key
```

Comment:
This example is not necessary, since you've already provided the commands to use to extract the cert above.

Reply:
agree